emission/tests/analysisTests/TestSimilarity.py

```diff
@@ -86,10 +94,10 @@ def testDeleteBins(self):
             self.assertTrue(b == sim.num)
 
     def testElbowDistance(self):
-        start = Coordinate(47,-122)
-        end = Coordinate(47,-123)
-        now = datetime.datetime.now()
-        t = Trip(None, None, None, None, now, now, start, end)
+        start = [-122,47]
+        end = [-133,47]
```

Comment:
Typo. Will fix along with anything else that you catch.
Also, note that I didn't write the original tests - they were written by a SUPERB student from the last year. So you too can write tests :)